store/copr/coprocessor.go

```diff
@@ -66,6 +67,8 @@ var (
 const (
 	copBuildTaskMaxBackoff = 5000
 	copNextMaxBackoff      = 20000
+	copSmallTaskRow        = 6
```

Comment:
Is this number a bit small?

Reply:
Maybe it's difficult to set it to a certain value, we could add some comments about the scenarios that may benefit from this value and some others may not.

Reply:
What about `32`, the initial batch size of TiKV.

Reply:
It's fine to me.